include/slang/syntax/SyntaxNode.h

```diff
@@ -232,18 +241,15 @@ class SeparatedSyntaxList : public SyntaxListBase {
     public:
         using ParentList = std::conditional_t<std::is_const_v<std::remove_pointer_t<U>>,
                                               const SeparatedSyntaxList, SeparatedSyntaxList>;
-        iterator_base(ParentList& list, size_t index) : list(&list), index(index) {}
-        iterator_base(const iterator_base& other) = default;
-
-        iterator_base& operator=(const iterator_base& other) = default;
```

Comment:
Why did you change this? iterators are required to be copy assignable so removing this doesn't seem like a good idea.

Reply:
It seems identical to master branch.

Reply:
Why do you say it's identical? There's a diff showing here of removing the copy assignment operator. Perhaps you're working off an old checkout and need to rebase?

Reply:
That's my mistake, I would fix it.